sql/core/src/test/scala/org/apache/spark/sql/columnar/ColumnStatsSuite.scala

```diff
@@ -22,19 +22,19 @@ import org.apache.spark.sql.catalyst.expressions.InternalRow
 import org.apache.spark.sql.types._
 
 class ColumnStatsSuite extends SparkFunSuite {
+  testColumnStats(classOf[BooleanColumnStats], BOOLEAN, InternalRow(true, false, 0))
```

Comment:
Yes we can. `BINARY` was designed to extend from `ByteArrayColumnType` to avoid duplicate code since `BINARY` and `GENERIC` are quite similar.